utils/HACKING.adoc

```diff
@@ -52,4 +52,16 @@ tools to test the new release, and if you update *after* that you risk
 breaking them again without them noticing.
 
 For example, the magic numbers for 4.13 were updated in
-  dd7927e156b7cb2f9
+  https://github.com/ocaml/ocaml/commit/dd7927e156b7cb2f9cb73d2d54a15a9c8192139\2[dd7927e]
+
+Beware, though, that https://github.com/ocaml/ocaml/pull/12652/[PR #12652],
+which is more recent than the commit mentioned above, changes the
+way magic numbers are defined.
+
+To bump the magic numbers to version xyz simply run:
```

Comment:
I would propose to merge the previous paragraph with this sentence, while removing the `Beware` warning:

> Since https://github.com/ocaml/ocaml/pull/12652/[PR #12652], to bump the magic numbers to version xyz simply run: